presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -243,7 +248,7 @@ public Object optimize(SymbolResolver inputs)
         return visitor.process(expression, inputs);
     }
 
-    private static class ConstantExpressionVerifierVisitor
+    public static class ConstantExpressionVerifierVisitor
```

Comment:
Instead of exposing this class, add a method to ensure an expression is constant.